commons/ava_default_testnet/default_testnet_config.go

```diff
@@ -0,0 +1,73 @@
+package ava_default_testnet
+
+/*
+	For the default testnet, there are five bootstrappers. They have hardcoded bootstrapperIDs that correspond to their TLS certs.
+	This must be hardcoded because Gecko requires specifying the bootstrapperID
+	along with the bootstrapperIP when connecting to bootstrappers in TLS mode.
+
+	The hardcoded IDs in this file are the known Gecko ID for a node using the private key and cert in this file.
+	They are also hardcoded in the gecko source code as the IDs for the initial stakers of the default testnet.
+*/
+
+type TestNet struct {
+	DefaultStakers []StakerIdentity
+	FundedAddresses FundedAddress
+}
+
+type FundedAddress struct {
+	Address string
+	PrivateKey string
+}
+
+type StakerIdentity struct {
+	NodeID string
+	PrivateKey string
+	TlsCert string
+}
+
+var DefaultTestNet = TestNet{
+	DefaultStakers: defaultStakers,
+	// hardcoded in Gecko in "genesis/config.go". needed to distribute genesis funds in tests
+	FundedAddresses: FundedAddress{
```

Comment:
How exactly do these FundedAddresses work?

Reply:
they're the accounts that are by default funded in the genesis block configuration (hardcoded in gecko codebase)